src/providers/index.tsx

```
@@ -1,22 +1,39 @@
+
+export function RootProvider({ children }: ProviderProps) {
 	return (
 		<>
-			<CookiesProvider>
-				<ReactQueryProvider>{children}</ReactQueryProvider>
-			</CookiesProvider>
-
-			<TanStackRouterDevtools position="bottom-left" />
+			{providers.reduceRight(
+				(acc, Provider, index) => (
+					<Provider key={index + 1}>{acc}</Provider>
```

Biome (2.4.4)

[error] Avoid using the index of an array as key property in an element.

(lint/suspicious/noArrayIndexKey)